_data/simple-icons.json

```diff
@@ -6440,6 +6440,11 @@
             "hex": "ED1C24",
             "source": "https://trakt.tv"
         },
+        {
+            "title": "TransferWise",
+            "hex": "00B9FF",
+            "source": "https://transferwise.com"
```

Comment:
Good job, thanks!
One last thing would be to change the source to https://brand.transferwise.com/logo and it will be ready to merge :)